projects/elife.yaml

```diff
@@ -361,6 +362,14 @@ api-gateway:
                 subdomains:
                     - "{instance}--cdn-gateway"
                     - api
+        test:
+            fastly:
+                subdomains:
+                    - "{instance}--cdn-gateway"
+                gcslogging:
+                    bucket: end2end-elife-fastly
```

Comment:
see https://console.cloud.google.com/storage/browser/end2end-elife-fastly/api-gateway--test/?project=elife-fastly&authuser=1&folder=true&organizationId=567577722038

Reply:
This is using the default format for logs, but I will integrate this:
https://docs.fastly.com/guides/streaming-logs/log-streaming-google-bigquery#example-format